vendor/github.com/OpenBazaar/go-ethwallet/wallet/wallet.go

```diff
@@ -809,16 +706,22 @@ func (wallet *EthereumWallet) CheckTxnRcpt(hash *common.Hash, data []byte) (*com
 			if err != nil {
 				return nil, err
 			}
-			wallet.db.Txns().Delete(chash)
-			toAddr := common.HexToAddress(util.EnsureCorrectPrefix(pTxn.To))
+			err = wallet.db.Txns().Delete(chash)
```

Comment:
It looks like the arg for `Delete` is generated out of `util.CreateChainHash((*hash).Hex())` but the arg which `Delete` is matching on is generated out of `ut.NormalizeAddress(hash.Hex())`. Are these going to produce the same value so that the Putted data will actually get Deleted?

Reply:
We also completely ignore the `err` set here, so if the Delete fails (say, because the arg doesn't exist) we'd never know.

Reply:
The key for delete matches.

Reply:
Okay. Let's still handle the error, though.